buildflow/runtime/runner.py

```diff
@@ -72,6 +108,17 @@ def __new__(cls, *args, **kwargs):
         return cls._instance
 
     def run(self, num_replicas: int):
+        if self._enable_usage:
+            print(
+                'Usage stats collection is enabled. To disable add the flag: '
+                '`--disable_usage_stats`.')
+            response = requests.post(
+                'https://apis.launchflow.com/buildflow_usage',
+                data=json.dumps({'id': self._session_id}))
```

Comment:
optional idea: Maybe write as a Session dataclass object so its easy for us to add other fields later

Reply:
done.